docs/state-channels/fsm_responder.puml

```diff
@@ -0,0 +1,48 @@
+@startuml
```

Comment:
Would it be possible to automate as Makefile the generation of this plantuml file, and hook that to CI?